### ISSUE TITLE
listing all the models we have tested and why we chose use it

### ISSUE THREAD

**Issue by george-lhj** [george-lhj/Transfer_Learning_for_Stock_Prediction#31]
Before and After adding all Signatures
1) Ridge Classifier
	- directly predicts 0/1, doesn’t provide probability estimates which makes it difficult to tune/adjust 	weights when incorrect
	- bound to specific loss functions 
	- penalty term must be L2
2) Logistic Regression, grouped by sector, industry, etc
	- b/c of data imbalance between each group, we are overfitting on training model results
	- allows us to choose different loss functions
	- too many signature columns to fit in the regression models (multicollinearity issue) -> resolves issue and might perform better
3) Neural Network
4) Logistic Regression w Kmeans w signature kernel

next step:
-
